.github/CODEOWNERS

```diff
@@ -0,0 +1,30 @@
+# Content
+/src/content/**/* @hashicorp/education
+
+## Education Engineering Managers
+/src/content/consul/**/* @hashicorp/consul-education-approvers
+/src/content/nomad/**/* @topfunky
+/src/content/terraform/**/* @hashicorp/terraform-education-approvers
+/src/content/vault/**/* @hashicorp/vault-education-approvers
+/src/content/boundary/**/* @hashicorp/boundary-education-approvers
+
+## Certifications
+/src/content/certifications/**/* @lszpunar
+
+
+# Engineering
+
+/.github @hashicorp/dev-portal
+/src/**/* @hashicorp/dev-portal
```

Comment:
Maybe this is too broad for now? But i'd rather start with a wide net and then refine as needed so we don't let things slip, especially since the repo is open source.